src/test/kotlin/dev/stalla/builder/fake/podcast/FakePodcastBuilder.kt

```diff
@@ -114,7 +116,9 @@ internal class FakePodcastBuilder : FakeBuilder<Podcast>(), ProvidingPodcastBuil
 
     override fun equals(other: Any?): Boolean {
         if (this === other) return true
-        if (other !is FakePodcastBuilder) return false
+        if (javaClass != other?.javaClass) return false
```

Comment:
This is a weird change, did you use the same settings to regenerate it?

Reply:
Uh yeah, no changes to IJ that I remember, except some update that was available a while ago.

Reply:
Just noticed that the changes from #90 also have this style

Reply:
Odd. Oh well, as long as it works, I suppose. They must've changed templates in the Kotlin IJ plugin.